docs/fsharp-cheatsheet.md

```diff
@@ -122,9 +123,6 @@ The `let` keyword also defines named functions.
     let square x = x * x
     let print x = printfn "The number is: %d" x
 
-    let squareNegateThenPrint x =
```

Comment:
needs to mvoe to L129

Reply:
Sorry, I'm not understanding this comment. Here is lines 127 to 131:

```
127 Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to improve readability especially in unit testing:
128
129    let ``square, negate, then print`` x =
130       x |> square |> negate |> print
131
```

Reply:
was saying that there is/was `print (square (negate x))`, then `print x |> square |> negate`, then `print >> square >> negate`
And one of the fns has `'` (prime) naming as it was the second.

if you remove the very first one, then the motivaiton for the second is lost
so prob best to move it down and work it into that section

(and consider the flow onward to showing it with `>>`)